File: session-notes/2015-02-24/ruby-exercises/longest_string.rb
def longest_string(list)
  longest = list.first

  list.each do |str|
    longest = str if str.length > longest.length
  end

  longest
end

if __FILE__ == $0
  input = ["a", "zzzz", "c"]
  expected = 'zzzz'

  p longest_string(input) == expected
end

File: session-notes/2015-02-24/longest_string.js
var longest_string = function(list) {
  var longest = list[0];

  list.forEach(function(item) {
    if (item.length > longest.length) {
      longest = item;
    }
  });

  return longest;
};

var input = ["a", "zzzz", "c"];
var expected = 'zzzz';

console.log(longest_string(input) == expected);
